lib/bootloader_setup.pm

```diff
@@ -351,7 +351,9 @@ sub uefi_bootmenu_params {
         assert_screen "gfxpayload_changed", 10;
         # back to the entry position
         send_key "home";
-        for (1 .. 10) { send_key "down"; }
+        for (1 .. 6) { send_key "down"; }
+        # On Leap/SLE we need to move down
+        for (1 .. 4) { send_key "down" if !is_tumbleweed; }
```

Comment:
IMO cleaner to do nothing once, instead of four noops.

```suggestion
        unless (is_sle('<16') || is_leap('<16')) {
            for (1 .. 4) { send_key "down"; }
        }
```

Reply:
It would an `if` instead of `unless`, but I get the idea. I will fix it.

Reply:
Indeed